apps/balancer-tools/src/app/apr/(components)/BALPrice.tsx

```diff
@@ -6,10 +6,13 @@ import { Round } from "../(utils)/rounds";
 export default async function BALPrice({
   roundId,
 }: {
-  roundId: string | string[];
+  roundId?: string | string[];
```

Comment:
Did this arg became optional because of the changes on `apps/balancer-tools/src/app/apr/pool/[network]/[poolId]/page.tsx`?

Reply:
Yes -- if we don't use a round it will just assume the current time